src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/AnalyzerReference.xaml

```diff
@@ -7,7 +7,8 @@
     Description="Analyzer reference properties"
     xmlns="http://schemas.microsoft.com/build/2009/properties">
     <Rule.DataSource>
-        <DataSource Persistence="ProjectFile" ItemType="Analyzer" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext" />
+        <DataSource Persistence="ProjectFile" ItemType="Analyzer" HasConfigurationCondition="False" SourceOfDefaultValue="AfterContext"
```

Comment:
This will mean that when a project is opened we have to wait for a dt build before any nodes show up in soln explorer. That's how the dependencies node used to work and the experience felt very slow. So we changed nodes to show up immediately based on evaluation and then it's resolved data would come from a DT build to add more information.  Now in this case, it's different that we cannot know about the analyzers until a DT build is complete. One thing we could do is to have two rules - one for evaluation and one for DT build and so when analyzers are directly added they show up immediately and the ones coming from packages show up later. That would require state management to dedupe the ones coming from dt build with the ones coming from evaluation. As far as I've seen, analyzers are mostly distributed as nuget packages and not directly added to a project. There all this state management might be a bunch of extra work for not much gain. 

So I think i'm okay with this approach. @davkean - any concerns?